src/core/lib/math/dftransform.cpp

```diff
@@ -211,18 +206,19 @@ std::vector<std::complex<double>> DiscreteFourierTransform::InverseTransform(std
     return invDftRemainder;
 }
 
-void DiscreteFourierTransform::FFTSpecialInvLazy(std::vector<std::complex<double>>& vals) {
+void DiscreteFourierTransform::FFTSpecialInvLazy(std::vector<std::complex<double>>& vals,
```

Comment:
Why is prepValues needed? Why not read it from the static map associated with the class?